packages/shorebird_cli/lib/src/executables/powershell.dart

```diff
@@ -19,7 +19,6 @@ class Powershell {
   Future<ShorebirdProcessResult> pwsh(
     List<String> arguments, {
     String? workingDirectory,
-    bool runInShell = false,
```

Comment:
This was never used so removing for now.

Reply:
An in general it's not something you ever want. :)